src/OrchardCore.Modules/OrchardCore.Media/Startup.cs

```diff
@@ -190,7 +191,14 @@ public override void Configure(IApplicationBuilder app, IRouteBuilder routes, IS
             {
                 // The tenant's prefix is already implied by the infrastructure
                 RequestPath = AssetsUrlPrefix,
-                FileProvider = new PhysicalFileProvider(mediaPath)
+                FileProvider = new PhysicalFileProvider(mediaPath),
+                ServeUnknownFileTypes = true,
+
+                // Cache the media files for 7 days
+                OnPrepareResponse = ctx =>
+                {
```

Comment:
Maybe only if we are not in "Development".

One question, what happens if we use e.g `OC.Media.Azure`?

Reply:
Okay i understand